src/database/schema.rs

```diff
@@ -1,34 +1,34 @@
 table! {
     bundle (id) {
-        id -> Bpchar,
-        owner_address -> Nullable<Bpchar>,
-        block_height -> Int8,
+        id -> Nullable<Text>,
+        owner_address -> Nullable<Text>,
```

Comment:
Not exactly related to this PR, but I would expect this column not to allow `NULL`? How can be have bundle without owner?

Reply:
Should not be nullable anymore. Weren't these regenerated?